File: String/Palindrome/C/isPalindrome.c

#include <stdio.h> 
#include <string.h> 

#define int long long int
#undef int

int isPalindrome(char str[]) 
{ 
    #define int long long int

    int l = 0; 
    int h = strlen(str) - 1; 

    while (h > l) 
    { 
        if (str[l++] != str[h--]) 
        { 
            printf("%s is Not Palindrome\n", str); 
            return; 
        } 
    } 
    printf("%s is palindrome\n", str); 
} 
  
 
int main() 
{ 
    #define int long long int

    char s[105];
    scanf("%s", s);

    isPalindrome(s);

    return 0; 
}

File: String/Palindrome/Javascript/isPalindrome.js
 let palindrome = (str) => {
      str = str.toLowerCase().replace(/[\W_]/g, '');
      for(let i = 0, len = str.length - 1; i < len/2; i++) {
        if(str[i] !== str[len-i]) {
          return false;
        }
      }
      return true;
    }
